api/app/controllers/api/v1/statistics_controller.rb

```diff
@@ -0,0 +1,16 @@
+module Api
+  module V1
+    # Statistics controller
+    class StatisticsController < ApplicationController
+
+      resourceful! Statistics, authorize_options: { except: [:index] } do
```

Comment:
And there should be a test for the index route showing that a non-admin user can't access it, and that an admin user can.